opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -217,10 +228,39 @@ def export(
             # Cloud Monitoring API allows, for any combination of labels and
             # metric name, one update per WRITE_INTERVAL seconds
             updated_key = (metric_descriptor.type, record.labels)
-            last_updated_seconds = self._last_updated.get(updated_key, 0)
-            if seconds <= last_updated_seconds + WRITE_INTERVAL:
+            last_updated_time = self._last_updated.get(updated_key, 0)
+            last_updated_time_seconds = last_updated_time // 1e9
+            if seconds <= last_updated_time_seconds + WRITE_INTERVAL:
                 continue
-            self._last_updated[updated_key] = seconds
+
+            if (
+                metric_descriptor.metric_kind
+                == MetricDescriptor.MetricKind.CUMULATIVE
+            ):
+                if (
+                    instrument.meter.batcher.stateful
+                    or updated_key not in self._last_updated
+                ):
+                    # The aggregation has not reset since the exporter has started up, so that is the start time
+                    point.interval.start_time.seconds = int(
+                        self._exporter_start_time_seconds
+                    )
+                    point.interval.start_time.nanos = int(
+                        self._exporter_start_time_nanos
+                    )
+                else:
+                    # The aggregation reset the last time it was exported
+                    (
+                        point.interval.start_time.seconds,
+                        point.interval.start_time.nanos,
+                    ) = map(
+                        int, divmod(self._last_updated[updated_key] + 1e6, 1e9)
```

Comment:
Would be good to have a comment and/or link to those docs here about the 1ms.

Reply:
added